ENIGMAsystem/SHELL/Bridges/General/DX11Context.h

```diff
@@ -113,15 +111,11 @@ int GetShapesModel() {
 void BeginShapesBatching(int texId) {
```

Comment:
It's absolutely worth extracting, and I already intended to do so. These 3 files here need deleted, and I intend to delete them all. But no, I will not do it in this pr because I do not want to conflate issues.